tests/unit_tests/test_wave_06.py

```diff
@@ -201,9 +201,10 @@ def test_swap_best_by_category_no_match_is_false():
     )
 
     assert result == False
-    assert tai.inventory[0].get_category() != "Clothing"
-    assert tai.inventory[1].get_category() != "Clothing"
-    assert tai.inventory[2].get_category() != "Clothing"
+    assert len(tai.inventory) == 3
+    assert len(jesse.inventory) == 3
```

Comment:
The length checks here are redundant. The list comparisons could not pass if the lists differed in length.